### ISSUE TITLE
How can I filter warnings?

### ISSUE THREAD

**Issue by ckemere** [nelpy/nelpy#299]
For example, we'll often see the warning: `WARNING:root:ignoring signal outside of support` when working with complex epochs. I know that there's signal outside of the support - I'm using the epoch to isolate it! I'd love to filter this warning out in that particular code block, but because it's using `logging` rather than `warning` filtering does not seem to be available.

**Comment by jchutrue** [comment]
I haven't tested this out myself, but try importing the logging module before nelpy, setting the desired level e.g.

```
import logging
logging.basicConfig()
logging.getLogger().setLevel(logging.ERROR)
```

and seeing if that filters out the warning. This snippet applies the settings globally, however. If you want something more selective we can look into deeper into this.

**Comment by shayokdutta** [comment]
> ```
> import logging
> logging.basicConfig()
> logging.getLogger().setLevel(logging.ERROR)
> 
> import nelpy as nel
> ...
> ```

This does work. Exactly the "work around" I've been using

**Comment by ckemere** [comment]
Yep. This makes sense. I guess the question is why we choose to use `logging` rather than `warning`, which can be filtered...

**Comment by jchutrue** [comment]
Partly documented in https://github.com/nelpy/nelpy/issues/76

@shayokdutta Really we should be using a different logger for each class rather than the global root logger.
